docs/app_examples.md

```diff
@@ -238,6 +238,18 @@ This script is intended as a drop-in replacement with the same syntax. (It can b
 | Do        | @code{}sh -c "kscreen-doctor output.HDMI-A-1.mode.${SUNSHINE_CLIENT_WIDTH}x${SUNSHINE_CLIENT_HEIGHT}@${SUNSHINE_CLIENT_FPS}"@endcode |
 | Undo      | @code{}kscreen-doctor output.HDMI-A-1.mode.3840x2160@120@endcode                                                                     |
 
+@attention{The names of your displays will differ between X11 and Wayland.
+Be sure to use the correct name, depending on your session manager.
+e.g. On X11, the monitor may be called HDMI-A-0, but on Wayland, it may be called "HDMI-A-1".
+}
+
+@hint{Replace "HDMI-A-1 with the display name of the monitor you would like to use for Moonlight.
+You can list the monitors available to you with `kscreen-doctor -o`.
+These will also give you the supported display properties for each monitor. You can select them either by
+hard-coding them their corresponding number (e.g. `kscreen-doctor output.HDMI-A1.mode.0`) or using the above
+"do" command to fetch the resolution requested by your Moonlight client
+(which has a chance of not being supported by your monitor).
+}
```

Comment:
![image](https://github.com/user-attachments/assets/ef1346bd-79fa-4ba3-b053-ccc41bb52921)

I guess the inline code blocks don't work when inside of an admonition. Try using `@code` and `@endcode`.

Reply:
If you don't figure this out, I can try to find the correct example in a couple of hours.

Reply:
I found some examples just by scrolling up, didn't realize nothing needed to actually be in the @code{} brackets, just making sure they work when the next doc is generated.

Thanks for the pointer on the readthedocs build. I'm still quite new to github and did not know that was a thing!

Reply:
I think you have the right syntax now, but might have accidentally deleted the end of a code block. At L868 in https://github.com/LizardByte/Sunshine/pull/3287/commits/74c7e75c1b2bea2f4f928e3f65e3e4a144a512fc

This is doxygen syntax, and somethings you have to fight to get it to work.

Reply:
Thanks, I fixed it. I'm going to wait for the latest readthedocs build to generate with these changes and do a couple of proofreads.